lib/ignore/ignore.go

```diff
@@ -274,27 +284,28 @@ func hashPatterns(patterns []Pattern) string {
 	return fmt.Sprintf("%x", h.Sum(nil))
 }
 
-func loadIgnoreFile(file string, modtimes map[string]time.Time) ([]Pattern, error) {
+func loadIgnoreFile(file string, modtimes map[string]time.Time) ([]string, []Pattern, error) {
 	if _, ok := modtimes[file]; ok {
-		return nil, fmt.Errorf("Multiple include of ignore file %q", file)
+		return nil, nil, fmt.Errorf("Multiple include of ignore file %q", file)
```

Comment:
Nitpick, errors should not begin with a capital letter (because they'll get logged in the end like `l.Warnln("Couldn't load ignores: %v", err)` and we don't want a spurious mid-sentence capital).